src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -81,6 +81,7 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
         $container->setParameter('contao.image.valid_extensions', $mergedConfig['image']['valid_extensions']);
         $container->setParameter('contao.image.imagine_options', $mergedConfig['image']['imagine_options']);
         $container->setParameter('contao.image.reject_large_uploads', $mergedConfig['image']['reject_large_uploads']);
+        $container->setParameter('contao.authentication.2fa.enforce_backend', $mergedConfig['authentication']['2fa']['enforce_backend']);
```

Comment:
Shouldn't we re-use the `contao.security` domain here?

Reply:
We could. I've talked about the naming of this with @Toflar.

Reply:
We should. 😄